pkg/errors/types.go

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Mainflux
+// SPDX-License-Identifier: Apache-2.0
+
+package errors
```

Comment:
Consider moving HTTP errors to `pkg/http/errors.go`

Reply:
This package is supposed to contain all types of errors not only http